docs/en/solutions/apply-catboost-model.md

```diff
@@ -0,0 +1,212 @@
+
+Before applying a CatBoost model:
+
+**1.** Pull the [Docker image](https://hub.docker.com/r/yandex/tutorial-catboost-clickhouse) from the registry:
+
+```bash
+$ docker pull yandex/tutorial-catboost-clickhouse
+```
+
+This Docker image contains everything you need to run an application: code, runtime, libraries, environment variables, and configuration files.
+
+**3.** Make sure the Docker image has been pulled:
+
+```bash
+$ docker image ls
+REPOSITORY                            TAG                 IMAGE ID            CREATED             SIZE
+yandex/tutorial-catboost-clickhouse   latest              3e5ad9fae997        19 months ago       1.58GB
+```
+
+**2.** Start the Docker-configured image:
+
+```bash
+$ docker run -it -p 8888:8888 yandex/tutorial-catboost-clickhouse
+```
+
+> **Note:** Example running a Jupyter Notebook with this manual materials to [http://localhost:8888](http://localhost:8888).
+
+## 1. Create a table {#create-a-table}
+
+To create a ClickHouse table for the train sample:
+
+**1.** Start a ClickHouse client:
+
+```bash
+$ clickhouse client
+```
+
+> **Note:** The ClickHouse server is already running inside the Docker container.
+
+**2.** Create the table using the command:
+
+```sql
+:) CREATE TABLE amazon_train
+(
+    date Date MATERIALIZED today(), 
+    ACTION UInt8, 
+    RESOURCE UInt32, 
+    MGR_ID UInt32, 
+    ROLE_ROLLUP_1 UInt32, 
+    ROLE_ROLLUP_2 UInt32, 
+    ROLE_DEPTNAME UInt32, 
+    ROLE_TITLE UInt32, 
+    ROLE_FAMILY_DESC UInt32, 
+    ROLE_FAMILY UInt32, 
+    ROLE_CODE UInt32
+)
+ENGINE = MergeTree(date, date, 8192)
+```
+
+## 2. Insert the data to the table {#insert-the-data-to-the-table}
+
+To insert the data:
+
+**1.** Exit from ClickHouse:
+
+```sql
+:) exit
+```
+
+**2.** Upload the data:
+
+```bash
+$ clickhouse client --host 127.0.0.1 --query 'INSERT INTO amazon_train FORMAT CSVWithNames' < ~/amazon/train.csv
+```
+
+**3.** Make sure the data has been uploaded:
+
+```sql
+$ clickhouse client
+:) SELECT count() FROM amazon_train
+SELECT count()
+FROM amazon_train
+
++-count()-+
+|   32769 |
++---------+
+```
+
+## 3. Configure the model to work with the trained model {#configure-the-model}
+
+This step is optional: the Docker container contains all configuration files. 
+
+**1.** Create a config file (for example, `config_model.xml`) with the model configuration:
+
+```xml
+<models>
+    <model>
+        <!-- Model type. Now catboost only. -->
+        <type>catboost</type>
+        <!-- Model name. -->
+        <name>amazon</name>
+        <!-- Path to trained model. -->
+        <path>/home/catboost/tutorial/catboost_model.bin</path>
+        <!-- Update interval. -->
+        <lifetime>0</lifetime>
+    </model>
+</models>
+```
+
+> **Note:** To show contents of the config file in the Docker container, run `cat models/amazon_model.xml`.
+
+**2.** Add the following lines to the `/etc/clickhouse-server/config.xml` file:
```

Comment:
usually we recommend not to do this in favor of /etc/clickhouse-server/config.d/ ?